R/theme_map.R

```diff
@@ -61,8 +66,8 @@ theme_map <- function(map = NULL, continuous = FALSE,
     map <- map +
       ggplot2::guides(fill = ggplot2::guide_legend(title = variable_label, ncol = 2)) +
       scale_fill(
-        # values = values,
-        palette = palette,
+        values = values,
+        # palette = palette,
```

Comment:
remove - not comment?

Reply:
Done in https://github.com/epiforecasts/EpiNow/pull/61/commits/f5515b5f1c5e59e529c0c85acc3844abf1b70769